tests/legacy-cli/e2e/tests/i18n/legacy.ts

```diff
@@ -226,7 +238,10 @@ export async function setupI18nConfig(useLocalize = true, format: keyof typeof f
   const translationFile = `src/locale/messages.${formats[format].ext}`;
   await expectFileToExist(translationFile);
   await expectFileToMatch(translationFile, formats[format].sourceCheck);
-  await expectFileToMatch(translationFile, `An introduction header for this sample`);
+
+  if (format !== 'json') {
```

Comment:
@clydin @petebacondarwin The message description will not be extracted into JSON files.